bk-config/bin/create_registration_cert.sh

```diff
@@ -4,23 +4,51 @@
 
 if [ $# -eq 0 ]
   then
-    echo "Usage: create_registration_cert.sh MINUTES"
+    echo "Usage: create_registration_cert.sh VALID"
     echo ""
-    echo "    MINUTES: the length of time (in minutes) the certificate is valid for"
+    echo "    the length of time the certificate is valid for"
+    echo "    examples: +3m (3 minutes) , +52w (52 weeks) , forever"
     exit 1
 fi
 
 
+if [ "${1}_" == "_"  ] ; then
+  echo "argument missing"
+  exit 1
+fi
+
+VALID=""
+if [ "${1}_" != "forever_"  ] ; then
+  VALID="-V ${1}"
+fi
+
+
+
+
 set -e
 set -x
 
 mkdir -p /sage_temporary
 cp /usr/lib/sage/registration_keys/id_rsa_sage_registration.pub /sage_temporary/
 
 
-ssh-keygen -I sage_registration -s /usr/lib/sage/certca/sage_beekeeper_ca -n sage_registration -V +${1}m -O no-agent-forwarding -O no-port-forwarding -O no-pty -O no-user-rc -O no-x11-forwarding -O force-command=/opt/sage/beekeeper/register/register.sh /sage_temporary/id_rsa_sage_registration.pub
+ssh-keygen -I sage_registration -s /usr/lib/sage/certca/sage_beekeeper_ca -n sage_registration ${VALID} -O no-agent-forwarding -O no-port-forwarding -O no-pty -O no-user-rc -O no-x11-forwarding -O force-command=/opt/sage/beekeeper/register/register.sh /sage_temporary/id_rsa_sage_registration.pub
```

Comment:
you use this path `/sage_temporary/id_rsa_sage_registration.pub` a few different places.  make it a variable?